app/helpers/mailer_helper.rb

```diff
@@ -8,8 +8,12 @@ def tool_name
     @tool_name ||= ApplicationService.application_name
   end
 
-  def helpdesk_email
-    @helpdesk_email ||= Rails.configuration.x.organisation.helpdesk_email
+  def helpdesk_email(org = nil)
```

Comment:
prefer named attribute and no if/else block since we don't need to make it an instance variable here, that is being done in the mailer. Something like:
```
def helpdesk_email(org: nil)
  org&.helpdesk_email || Rails.configuration.x.organisation.helpdesk_email
end
```

Reply:
Updated as suggested.